ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -98,18 +96,25 @@ export class ScenePlayerImpl extends React.Component<
 
     this.player.on("error", (err: any) => {
       if (err && err.code === 224003) {
-        this.handleError();
+        // When jwplayer has been requested to play but the browser doesn't support the video format.
+        this.handleError(true);
       }
     });
 
+    //
     this.player.on("meta", (metadata: any) => {
       if (
         metadata.metadataType === "media" &&
         !metadata.width &&
         !metadata.height
       ) {
-        // treat this as a decoding error and try the next source
-        this.handleError();
+        // Occurs during preload when videos with supported audio/unsupported video are preloaded.
+        // Treat this as a decoding error and try the next source without playing.
+        // However on Safari we get an media event when m3u8 is loaded which needs to be ignored.
+        const currentFile = this.player.getPlaylistItem().file;
+        if (currentFile != null && !currentFile.includes("m3u8")) {
+          this.handleError(false);
```

Comment:
```suggestion
          const state = this.player.getState();
          const play = state === "buffering" || state === "playing";
          this.handleError(play);
```

This fixes the issue of needing to click the play button multiple times.